opendevin/llm/llm.py

```diff
@@ -91,6 +92,9 @@ def __init__(
                 # Max output tokens for gpt3.5, so this is a safe fallback for any potentially viable model
                 self.config.max_output_tokens = 1024
 
+        if self.config.drop_params:
+            litellm.drop_params = self.config.drop_params
```

Comment:
This should work. It seems also possible that litellm accepts it in the completion call 🤔, according to their latest code. (the two links to litellm in a comment above have had some changes two weeks ago and they are sending it in `completion` at least to openai/azure)